contentcuration/contentcuration/views/base.py

```diff
@@ -168,16 +173,20 @@ def publish_channel(request):
         raise ObjectDoesNotExist("Missing attribute from data: {}".format(data))
 
 
-class SQCountDistinct(Subquery):
+class SQCount(Subquery):
     # Include ALIAS at the end to support Postgres
-    template = "(SELECT COUNT(DISTINCT %(field)s) FROM (%(subquery)s) AS %(field)s__sum)"
```

Comment:
Motivation for changing this to a non-distinct count? This will massively inflate the number of resources in the case of KA.

Reply:
This might have been from a merge conflict